inference/generativeai/llm-workshop/flan-ul2-pySDK/flan-ul2-pySDK.ipynb

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
<p>Is this approved by legal?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4134/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>